Mathlib/Analysis/Complex/Polynomial/Basic.lean

```diff
@@ -47,6 +48,10 @@ instance isAlgClosed : IsAlgClosed ℂ :=
 
 end Complex
 
+theorem Real.nonempty_algEquiv_or (F : Type*) [Field F] [Algebra ℝ F] [Algebra.IsAlgebraic ℝ F] :
```

Comment:
This one is probably worth attaching a docstring to.

Reply:
```suggestion
/-- An algebraic extension of ℝ is isomorphic to either ℝ or ℂ as an ℝ-algebra. -/
theorem Real.nonempty_algEquiv_or (F : Type*) [Field F] [Algebra ℝ F] [Algebra.IsAlgebraic ℝ F] :
```